.github/workflows/manual-publish.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       id-token: write
       contents: write
     steps:
-      - uses: ./.github/workflows/full-release.yml
```

Comment:
I don't know if this applies to this one, but the dotnet-client has jobs in making a full release, so it shares the workflow. It only does because different sub-jobs need different runners. That shouldn't apply to that case. But I wanted to note it before signing this off.

Reply:
I see it doesn't do that, so this must be a copy/paste error.

Reply:
Whoops! My b.